packages/subgraph/src/plugin/plugin.ts

```diff
@@ -67,7 +75,7 @@ export function handleProposalCreated(event: ProposalCreated): void {
 
       const actionId = generateActionEntityId(
         pluginAddress,
-        Address.fromBytes(daoAddr),
+        Address.fromString(daoAddress),
```

Comment:
because of separating handleProposalCreated in: handleProposalCreated and _handleProposalCreated
The 1st one retrieves daoAddress as a string, whereas previously it already cast it as an address

Reply:
Conclusion: We shouldn't use Address.fromHexString(), fixed the above .fromHexString